src/software/ai/navigator/navigator_test.cpp

```diff
@@ -201,3 +203,109 @@ TEST(NavigatorTest, convert_multiple_intents_to_primitives)
     auto pivot_primitive = dynamic_cast<PivotPrimitive &>(*(primitive_ptrs.at(1)));
     EXPECT_EQ(expected_pivot_primitive, pivot_primitive);
 }
+
+TEST(NavigatorTest, move_intent_with_one_point_path_test_path_planner)
+{
+    // TODO: refactor this into the setup and constructor and add more of these types of
+    // tests
```

Comment:
TODO?

Reply:
Yeah it works but it needs a refactor when a complete set of tests are written for navigator.